packages/training/book/src/development/codebase-map.md

```
@@ -0,0 +1,296 @@
+# Codebase Map
+
+A guide to the key files and their purposes.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)